utils/rpms/packaging/Makefile_packaging.mk

```diff
@@ -39,7 +39,7 @@ EL_7_PR_REPOS            ?= $(shell git show -s --format=%B | sed -ne 's/^PR-rep
 UBUNTU_20_04_PR_REPOS    ?= $(shell git show -s --format=%B | sed -ne 's/^PR-repos-ubuntu20: *\(.*\)/\1/p')
 
 COMMON_RPM_ARGS  := --define "_topdir $$PWD/_topdir" $(BUILD_DEFINES)
-SPEC             := $(shell if [ -f $(NAME)-$(DISTRO_BASE).spec ]; then echo $(NAME)-$(DISTRO_BASE).spec; else echo $(NAME).spec; fi)
+SPEC             := $(shell if [ -f $(NAME)-$(DISTRO_BASE).spec ]; then echo $(NAME)-$(DISTRO_BASE).spec; else echo $(NAME)$(COVERAGE).spec; fi)
```

Comment:
This is supposed to be an **exact** copy of https://github.com/daos-stack/daos/blob/master/utils/rpms/packaging/Makefile_packaging.mk as it will be overwritten by just copying from that file at any time.  This is also something we want to maintain a fork of.